### ISSUE TITLE
[CI] Failing Check

### ISSUE THREAD

**Issue by abebeos** [SecureSECO/SearchSECOController#61]
related: #

From https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492
```sh
Run /controller/build/searchseco checkupload https://github.com/SecureSECO/SearchSECOController -b 60/merge
2022-10-2[4](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:5) 04:38:03.819 (   0.007s) [controller      ]           commands.cpp:80    INFO| Checking the code from the project at https://github.com/SecureSECO/SearchSECOController against the SearchSECO database
error: pathspec '60/merge' did not match any file(s) known to git
error: pathspec '60/merge' did not match any file(s) known to git
2022-10-24 04:38:07.499 (   3.686s) [parser          ]             Parser.cpp:27    INFO| Starting Parser
2022-10-24 04:38:07.499 (   3.687s) [parser          ]             Parser.cpp:32    INFO| Parsing [5](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:6)2 files
2022-10-24 04:38:07.798 (   3.985s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 9%.
2022-10-24 04:38:07.897 (   4.084s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 19%.
2022-10-24 04:38:07.935 (   4.122s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 28%.
2022-10-24 04:38:08.023 (   4.210s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 38%.
2022-10-24 04:38:08.109 (   4.297s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 48%.
2022-10-24 04:38:08.122 (   4.309s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 57%.
2022-10-24 04:38:08.150 (   4.337s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: [6](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:7)[7](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:8)%.
2022-10-24 04:3[8](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:9):08.202 (   4.38[9](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:10)s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 76%.
2022-[10](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:11)-24 04:38:08.232 (   4.419s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 86%.
2022-10-24 04:38:08.244 (   4.432s) [parser          ]          XmlParser.cpp:175   INFO| Parsing: 96%.
2022-10-24 04:38:08.252 (   4.440s) [parser          ]             Parser.cpp:77    INFO| Parsing finished, total number of methods found: 157
2022-10-24 04:38:08.253 (   4.440s) [controller      ]          GitSpider.cpp:77    INFO| Blaming and processing 52 files
2022-10-24 04:38:09.038 (   5.225s) [controller      ]   databaseRequests.cpp:268    ERR| E403 - Database responded in an unexpected way. Please try again later.
Error: Process completed with exit code [14](https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5462940492#step:3:15)7.
```

## Workaround:

- disable for now
- 
## Related / Duplicate

- #66

**Comment by abebeos** [comment]
Disabled via github UI: https://github.com/SecureSECO/SearchSECOController/actions/workflows/checking_example.yml

**Comment by slingerbv** [comment]
The DB is getting increasingly annoying. Can't wait to find out why it's giving the "unexpected" response.

**Comment by abebeos** [comment]
Hey! No db here!

**Comment by abebeos** [comment]
@slingerbv , db problems stuff comes up here: #63 (but be patient, this can take days or even weeks, depending on how many issues "pop up").

**Comment by abebeos** [comment]
> Hey! No db here!

My fault, you actually looked at the error log. Bug was fixed within #66

**Comment by abebeos** [comment]
still fails, but at later point, with upload:

```
2022-10-26 11:23:22.779 (  59.845s) [controller      ]   databaseRequests.cpp:255    ERR| E402 - Something went wrong in the database. Following error occured in the database: Failed to add project to database.
```
from: https://github.com/SecureSECO/SearchSECOController/actions/runs/3309597685/jobs/5502971375

disabling this in CI again.

**Comment by abebeos** [comment]
closing this here, as original issue was fixed within #66.

Follow-up: #72
